apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -382,12 +413,12 @@ get_chunk_group_size() ->
 	{ok, Config} = application:get_env(arweave, config),
 	Config#config.chunk_storage_file_size.
 
-read_repack_cursor(StoreID) ->
-	Filepath = get_filepath("repack_cursor", StoreID),
+read_repack_cursor(StoreID, TargetPacking) ->
+	Filepath = get_filepath("repack_in_place_cursor", StoreID),
```

Comment:
the repack cursor is still used when repacking across modules as well as repacking in place right? (i.e. I shouldn't read to much into the `"repack_in_place_cursor"` name)

Reply:
Oh actually I should read into it. Looks like we're no longer loading the cursor from disk if we're repacking across storage_modules?

https://github.com/ArweaveTeam/arweave/pull/528/files#diff-70860f94e3504d0cdd89e58a65a090fadb6a75eabc58f0a89aeae829109e8dbbL241

Reply:
The cross-module repacking does not use a cursor. This here is a part of the spora_2_5 repacking process that was turned off at some point.

Reply:
ahh, got it. I was wondering why I didn't notice it when I had started looking into the repack resume bug. Makes sense.